### PR TITLE
Add library call to create DHCP with dns servers and host routes.

### PR DESCRIPTION
**Pull request by JoeMido**
- Added create_bridge_dhcp all that accepts host_rts and dns_servers.

Signed-off-by: Joe Mills joe@midokura.com

### CODE REVIEW

File: src/midonetclient/api.py [hunk 1]
```diff
@@ -28,6 +28,7 @@
 
 LOG = logging.getLogger(__name__)
 
+
```
Comment (rossella) [inline src/midonetclient/api.py:31]: Joe please remove the extra line

Comment (JoeMido) [inline src/midonetclient/api.py:31]: pep8 likes an extra line before the start of definitions in a python file. This was part of my attempt at satisfying pep8 for this commit.

File: src/midonetclient/api.py [hunk 2]
```diff
@@ -260,7 +289,7 @@ def add_router_route(self, router, type='Normal',
         route = router.add_route().type(type)
         route = route.src_network_addr(src_network_addr).src_network_length(
```
Comment (admytren) [inline src/midonetclient/api.py:290]: Ouch! Could this be broken up into a few assignments for better code readability :)

File: src/midonetclient/api.py [hunk 3]
```diff
@@ -184,6 +185,34 @@ def add_bridge(self):
...
 
+    def add_bridge_dhcp(self, bridge, gateway_ip, cidr, host_rts=None,
+                        dns_nservers=None):
+        if host_rts is None:
+            host_rts = []
+
+        if dns_nservers is None:
+            dns_nservers = []
+
+        net_addr, net_len = cidr.split('/')
```
Comment (admytren) [inline src/midonetclient/api.py:196]: Dumb question, is netmask always specified as an integer or could it be like '255.255.255.0'

**Comment by admytren** [comment]
Overall looks good!

**Comment by rossella** [comment]
@JoeMido can you answer Artem's questions then I'll merge your changes

**Comment by JoeMido** [comment]
These comments were addressed and the changes merged as a part of a new pull request:

https://github.com/midokura/python-midonetclient/pull/75

**Pull request closed without merging**